src/main/java/roomescape/repository/TimeRepository.java

```diff
@@ -0,0 +1,52 @@
+package roomescape.repository;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.TimeSlot;
+import roomescape.domain.TimeSlotDto;
+
+import java.util.List;
+
+@Repository
+public class TimeRepository {
```

Comment:
Dao 와 Repository 의 차이는 무엇일까요?

참고 : https://www.baeldung.com/java-dao-vs-repository

Reply:
> DAO는 데이터 액세스 로직을 추상화하고 직접적으로 데이터베이스와 상호작용하는 데 사용되는 반면,
Repository는 도메인 객체의 영속성을 관리하고 도메인과 데이터 액세스 계층 사이 CRUD 작업을 제공하는 데 사용된다고도 할 수 있을까요? 🤔

말씀하신대로, DAO 는 정말 DB 와의 상호 작용만을 담당한다면 repository 는 dao 를 활용해서 데이터를 불러와 예외처리를 할 수도 있고, 도메인으로 변경하여 반환하는 작업을 할 수도 있습니다.  

repository 가 dao 를 품고 있는 ? 그런 느낌이라고 이해하시면 좋을 것 같습니다!


참고 자료 중 일부 
> In other words, a repository also deals with data and hides queries similar to DAO. However, it sits at a higher level, closer to the business logic of an app.

Consequently, a repository can use a DAO to fetch data from the database and populate a domain object. Or, it can prepare the data from a domain object and send it to a storage system using a DAO for persistence.